packages/gatsby/src/commands/develop-process.ts

```diff
@@ -248,6 +248,7 @@ async function startServer(program: IDevelopArgs): Promise<IServer> {
     activity.start()
     await sourceNodes({
       webhookBody: req.body,
+      webhookRequest: req,
```

Comment:
Just one thing here - we were talking a bit that passing `req` as-is might be tricky for few reasons (we change some internal express configuration and now `request` has different structure).

So we will likely want to be explicit in what we pass down. In this case I think we would only want to have object with `body` and `headers` (and if there will be need for additional things - add it then):
```suggestion
      webhookRequest: {
        body: req.body,
        headers: req.headers,
      },
```

Reply:
I am OK with that, I just quickly checked other properties of the request and I haven't found anything interesting for my use cases right now. And this could be added later on :-)

Reply:
Only other thing that I could see useful potentially is the `query` or `params` (not sure what's it's called on express request from memory), if webhook would be fired using GET style params `<some_url>?param1=foo`, but this can be added if it's actually ever needed